src/Avalonia.X11/X11PlatformThreading.cs

```diff
@@ -235,10 +236,15 @@ public void RunLoop(CancellationToken cancellationToken)
                     return;
                 //Flush whatever requests were made to XServer
                 XFlush(_display);
-                epoll_event ev;
                 if (XPending(_display) == 0)
-                    epoll_wait(_epoll, &ev, 1,
-                        nextTick == null ? -1 : Math.Max(1, (int)(nextTick.Value - _clock.Elapsed).TotalMilliseconds));
+                {
+                    var epollWaitTimeout = nextTick == null ? -1 : Math.Max(1, (int)(nextTick.Value - _clock.Elapsed).TotalMilliseconds);
+                    var epollWaitStatus = epoll_wait(_epoll, out _, 1, epollWaitTimeout);
+                    if (epollWaitStatus == -1)
```

Comment:
@kekekeks I guess we might not want to throw if `epoll_wait` was interrupted by a signal handler, e.g. `SIGABRT` ?

Reply:
epoll_wait is used here only to save CPU cycles, so return value can be ignored.